pom.xml

```diff
@@ -71,8 +77,18 @@
             <version>1.5.1</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>junit</artifactId>
+            <version>1.19</version>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
-
+    <properties>
+        <jenkins.version>2.32</jenkins.version>
```

Comment:
Please remove the version as it would make this plugin incompatible with Blue Ocean which needs 2.7.1 or above.

If there is a good reason for doing so (for example, CSS or JavaScript needs Jenkins 2) then set this to 2.7.1.